.circleci/config.yml

```diff
@@ -15,9 +15,8 @@ jobs:
       - checkout
       - restore_cache: &restore_cache
           keys:
-            - v5-{{checksum ".circleci/config.yml"}}-{{ checksum "yarn.lock"}}
-            - v5-{{checksum ".circleci/config.yml"}}
-            - v5
+            - cache-{{checksum ".circleci/config.yml"}}-{{ checksum "yarn.lock"}}
```

Comment:
@RodEsp why are the versions being removed?

Reply:
Because since they're hard-coded they don't actually match up with any particular version of the `oclif` package or the tooling used in the CI build and therefore aren't relevant.

But maybe I'm missing something? Why should we leave them in?

Reply:
Maybe no longer needed, resolving.